platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileWriter.java

```diff
@@ -166,14 +175,37 @@ public static void writeSignedStateFilesToDirectory(
         writeSettingsUsed(directory, platformContext.getConfiguration());
 
         if (selfId != null) {
-            copyPreconsensusEventStreamFiles(
+            copyPreconsensusEventStreamFilesRetryOnFailure(
                     platformContext,
                     selfId,
                     directory,
                     signedState.getState().getPlatformState().getPlatformData().getMinimumGenerationNonAncient());
         }
     }
 
+    private static void copyPreconsensusEventStreamFilesRetryOnFailure(
+            @NonNull final PlatformContext platformContext,
+            @NonNull final NodeId selfId,
+            @NonNull final Path destinationDirectory,
+            final long minimumGenerationNonAncient)
+            throws IOException {
+
+        int triesRemaining = COPY_PCES_MAX_RETRIES;
+        while (triesRemaining-- > 0) {
+            try {
+                copyPreconsensusEventStreamFiles(
+                        platformContext, selfId, destinationDirectory, minimumGenerationNonAncient);
+                return;
+            } catch (final PreconsensusEventFileRenamed e) {
+                logger.error(
+                        EXCEPTION.getMarker(),
```

Comment:
A lot of tests will keep failing with this exception

Reply:
Oops, the intention was to only log this if we run out of tries. I've refactored this so that it will only log when the number of remaining tries is 0.